zerver/lib/markdown/tabbed_sections.py

```diff
@@ -73,6 +73,7 @@
     "not-stream": "From other views",
     "via-recent-topics": "Via recent topics",
     "via-left-sidebar": "Via left sidebar",
+    "null_tab": "None",
```

Comment:
Why is this line needed?

Reply:
In case of a tab_section with no tabs, currently a single tab with the name 'null_tab' gets added (See https://github.com/zulip/zulip/blob/main/zerver/lib/markdown/tabbed_sections.py#L100)


So, I added the display name 'None' for 'null_tab', to keep in line with the existing behavior. Otherwise, this would break "zerver.tests.test_templates.TemplateTestCase.test_markdown_tabbed_sections_extension"
( Specifically this line - https://github.com/zulip/zulip/blob/main/zerver/tests/test_templates.py#L82 )

Reply:
OK -- this is a prime example of something that should have a nice comment explaining the reason.

I also wonder if we that code path should instead be doing a tab with "Instructions for all platforms" as the label, or similar, rather than being an exception to this pattern.

Reply:
Do you mean at #L100 in tabbed_sections.py, we should do something like this instead ?
`{"tab_name": "instructions-for-all-platforms", "start": tab_section["start_tabs_index"]}`
And then add such a mapping in "TAB_DISPLAY_NAMES" ?
`"instructions-for-all-platforms": "Instructions for all platforms"`

Reply:
Also, should this be done in the same commit or in a separate commit  ?

Reply:
This sort of change should be a separate commit.